### PR TITLE
Update Local payment package name

### PR DESCRIPTION
**Pull request by tdchow**
### Summary of changes

 - Update package name to `com.braintreepayments.api.localpayment`

### Checklist

 - [x] Added a changelog entry
 - [ ] Relevant test coverage

### CODE REVIEW

File: LocalPayment/src/androidTest/java/com/braintreepayments/api/LocalPaymentClientTest.java [hunk 1]
```diff
@@ -7,6 +7,10 @@
 import androidx.test.core.app.ApplicationProvider;
```
Comment (saperi22) [inline LocalPayment/src/androidTest/java/com/braintreepayments/api/LocalPaymentClientTest.java:7]: This file and other test files should also change their package structure, correct?

File: LocalPayment/src/androidTest/java/com/braintreepayments/api/LocalPaymentClientTest.java [hunk 2]
```diff
@@ -7,6 +7,10 @@
...
 import androidx.test.internal.runner.junit4.AndroidJUnit4ClassRunner;
 
+import com.braintreepayments.api.localpayment.LocalPaymentAuthRequest;
```
Comment (saperi22) [inline LocalPayment/src/androidTest/java/com/braintreepayments/api/LocalPaymentClientTest.java:10]: these imports should be unnecessary as they are local to the package once tests are also moved

**Review by saperi22 — APPROVED** [review]

File: LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentLauncher.kt [hunk 3]
```diff
@@ -62,12 +65,12 @@ class LocalPaymentLauncher internal constructor(private val browserSwitchClient:
      * completing the authentication flow.
      */
     fun handleReturnToAppFromBrowser(
-        pendingRequest: LocalPaymentPendingRequest.Started,
-        intent: Intent
+            pendingRequest: LocalPaymentPendingRequest.Started,
+            intent: Intent
```
Comment (jaxdesmarais) [inline LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentLauncher.kt:69]: Not sure if the changes in this file were auto formatted but it looks super far in

File: LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentLauncher.kt [hunk 4]
```diff
@@ -62,12 +65,12 @@ class LocalPaymentLauncher internal constructor(private val browserSwitchClient:
...
     ): LocalPaymentAuthResult {
         return when (val result = browserSwitchClient.parseResult(pendingRequest.request, intent)) {
             is BrowserSwitchResult.Success -> LocalPaymentAuthResult.Success(
-                LocalPaymentAuthResultInfo(result.resultInfo)
+                    LocalPaymentAuthResultInfo(result.resultInfo)
```
Comment (jaxdesmarais) [inline LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentLauncher.kt:73]: Same here, not sure if this is expected 😅

**Review by jaxdesmarais — APPROVED** [review]

**Pull request merged**